src/sketchup-stl/exporter.rb

```diff
@@ -334,6 +358,21 @@ def self.do_options
         window.show
       end # do_options
 
+
+      # Main entry point via menu item.
+      # Display a message and exit if the model is empty, else
+      # show the export dialog.
+      def self.main
+        if Sketchup.active_model.active_entities.length == 0
+          msg = "SketchUp STL Exporter:\n\n" +
```

Comment:
Style guide says to avoid + to concat strings. I think `<<` is preferred over `+`, but I also think you can do something like this in Ruby:

```
str = "Hello"\
  "World"
```